WebAPI/Controllers/ParcelController.cs

```diff
@@ -34,5 +36,12 @@ public ActionResult<Parcel> Post(Parcel parcel)
             _logger.LogInformation("Received Post request at parcel\n"+parcel.ToString()); //testing purposes
             return Ok(parcel);
         }
+        
+        [HttpGet("parcel/{id}")]
+        public ActionResult<Parcel> GetParcel(int id)
+        {
+            _logger.LogInformation("Received GET request at parcel/"+id); //testing purposes
+            return Ok(repo.Where(p => p.Id == id));
```

Comment:
If You're playing with`Ok()`, then would have been nice to return "not Ok" when entry with given id doesn't exist